visual-tests/test-scripts/indicators/indicators-combined.js

```diff
@@ -0,0 +1,119 @@
+        chartLayout = fc.utilities.chartLayout(),
+        chartBuilder = fc.utilities.chartBuilder(chartLayout);
+
+    chart.call(chartBuilder);
+
+    // Create scale for x axis
+    var dateScale = d3.time.scale()
+        .domain(fc.utilities.extent(data, 'date'))
+        .range([0, chartLayout.getPlotAreaWidth()])
+        .nice();
+
+    // Create scale for y axis
+    var priceScale = fc.scale.linear()
+        .domain(fc.utilities.extent(data, ['high', 'low']))
+        .range([chartLayout.getPlotAreaHeight(), 0])
+        .nice();
+
+    // Create the axes
+    var dateAxis = d3.svg.axis()
+        .scale(dateScale)
+        .orient('bottom');
+
+    var priceAxis = d3.svg.axis()
+        .scale(priceScale)
+        .orient('right');
+
+    // Create the OHLC series
+    var ohlc = fc.series.ohlc()
+        .xScale(dateScale)
+        .yScale(priceScale);
+
+    // Create the Bollinger bands
+    var bollinger = fc.indicators.bollingerBands()
+        .xScale(dateScale)
+        .yScale(priceScale)
+        .movingAverage(4)
+        .standardDeviations(2);
+
+    // Create the moving average component
+    var movingAverage = fc.indicators.movingAverage()
+        .xScale(dateScale)
+        .yScale(priceScale)
+        .averagePoints(10);
+
+    // Add the axes to the chart
+    chartBuilder.setAxis('bottom', dateAxis);
+    chartBuilder.setAxis('right', priceAxis);
+
+    // Add components to plot area
+    chartBuilder.addToPlotArea([ohlc, bollinger, movingAverage]);
+
+    // Set data and render
+    chartBuilder.setData(data);
+    chartBuilder.render();
+
+    // Create the RSI chart
+    var rsiChart = d3.select('#indicators-combined-rsi'),
+        rsiLayout = fc.utilities.chartLayout().height(150);
+
+    rsiChart.call(rsiLayout);
+
+    // Create RSI scale for y axis
+    var percentageScale = fc.scale.linear()
+        .domain([0, 100]) // Perctange scale
+        .range([rsiLayout.getPlotAreaHeight(), 0])
+        .nice();
+
+    var percentageAxis = d3.svg.axis()
+        .scale(percentageScale)
+        .orient('right')
+        .ticks(5);
+
+    rsiLayout.getAxisContainer('bottom').call(dateAxis);
+    rsiLayout.getAxisContainer('right').call(percentageAxis);
+
+    // Create the RSI component
+    var rsi = fc.indicators.rsi()
+        .xScale(dateScale)
+        .yScale(percentageScale)
+        .lambda(0.94);
+
+    // Add it to the chart
+    rsiLayout.getPlotArea().append('g')
+        .attr('class', 'rsi')
+        .datum(data)
+        .call(rsi);
+
+    // Update the data and chart (1 second interval)
+    setInterval(function() {
+        data.push(dataGenerator.generate(1)[0]);
+        data.shift();
+
+        // Update main chart
+        dateScale.domain(fc.utilities.extent(data, 'date'));
+        priceScale.domain(fc.utilities.extent(data, ['high', 'low']));
+        chartBuilder.setData(data);
+        chartBuilder.render();
+
+        // Update RSI chart
+        var rsi = fc.indicators.rsi()
```

Comment:
There was a reason, but I've just realised it's invalid - my mistake! I'll update the test now, before the merge.